### ISSUE TITLE
Conflict - I AM A MISSION BUILDER FOR MILSIM, I NEED THIS SOLVED ASAP

### ISSUE THREAD

**Issue by top1gunn** [oOKexOo/AresModAchillesExpansion#29]
so i got the new version of achiles that i absolutely love but when i installed 0.4, i got in ggame and couldnt open zeus because it said that i ahd the regular ares on, i left and turned it off and booted back up, opened zeus once more withouth ares just achilles, and it said i still had ares on and my zeus wont open anymore and its becoming quite frustrating. i nedd help as soon as possible

**Comment by oOKexOo** [comment]
I guess you are referring to the error messages "Error: Please unload Ares Mod!" and "Ares Mod and Ares Mod - Achilles Expansion are standalone add-ons and are NOT compatible with each other!".
Those message are observed in the case you have a patch class that called “Ares”. This patch class is present in Ares and in older versions of Achilles (goes back to 0.0.2). The patch class Ares is obviously present in a mod you have loaded; otherwise you would not observe such an error. Achilles 0.0.3 and 0.0.4 do not contain such a patch class and throw an error if they detect such a patch class.

So in conclusion you did not tell me all the details about all you actually did. For instance, where did you download Achilles from?

**Comment by oOKexOo** [comment]
This issue is probably out of date. If this problem is still present, please open a new ticket.

**Comment by top1gunn** [comment]
yeah i just removed all zeus mods at the time and reloaded the mod and it
worked, and thanks for making such an amazing mod, makes my job 10000x
easier

**Comment by Nels0** [comment]
I'm just concerned about your milsim community at this point.

**Comment by top1gunn** [comment]
And why would that be

**Comment by BryanBoru** [comment]
@top1gunn I think @Nels0 might have meant he would close the issue assuming its fixed, but commented to get your attention, in case the unit was still stranded with the problem.

**Comment by top1gunn** [comment]
I assumed it was a subiminal insult
